airflow/ui/src/components/DataTable/DataTable.tsx

```diff
@@ -106,87 +110,34 @@ export const DataTable = <TData,>({
-      <ChakraTable colorScheme="blue">
-        <Thead bg={theadBg} position="sticky" top={0} zIndex={1}>
-          {table.getHeaderGroups().map((headerGroup) => (
-            <Tr key={headerGroup.id}>
-              {headerGroup.headers.map(
-                ({ colSpan, column, getContext, id, isPlaceholder }) => {
-                  const sort = column.getIsSorted();
-                  const canSort = column.getCanSort();
+  const display = displayMode === "card" && Boolean(cardDef) ? "card" : "table";
 
-                  return (
-                    <Th
-                      colSpan={colSpan}
-                      cursor={column.getCanSort() ? "pointer" : undefined}
-                      key={id}
-                      onClick={column.getToggleSortingHandler()}
-                      whiteSpace="nowrap"
-                    >
-                      {isPlaceholder ? undefined : (
-                        <>{flexRender(column.columnDef.header, getContext())}</>
-                      )}
-                      {canSort && sort === false ? (
-                        <TiArrowUnsorted
-                          aria-label="unsorted"
-                          size="1em"
-                          style={{ display: "inline" }}
-                        />
-                      ) : undefined}
-                      {canSort && sort !== false ? (
-                        sort === "desc" ? (
-                          <TiArrowSortedDown
-                            aria-label="sorted descending"
-                            size="1em"
-                            style={{ display: "inline" }}
-                          />
-                        ) : (
-                          <TiArrowSortedUp
-                            aria-label="sorted ascending"
-                            size="1em"
-                            style={{ display: "inline" }}
-                          />
-                        )
-                      ) : undefined}
-                    </Th>
-                  );
-                },
-              )}
-            </Tr>
-          ))}
-        </Thead>
-        <Tbody>
-          {table.getRowModel().rows.map((row) => (
-            <Fragment key={row.id}>
-              <Tr>
-                {/* first row is a normal row */}
-                {row.getVisibleCells().map((cell) => (
-                  <Td key={cell.id}>
-                    {flexRender(cell.column.columnDef.cell, cell.getContext())}
-                  </Td>
-                ))}
-              </Tr>
-              {row.getIsExpanded() && (
-                <Tr>
-                  {/* 2nd row is a custom 1 cell row */}
-                  <Td colSpan={row.getVisibleCells().length}>
-                    {renderSubComponent?.({ row })}
-                  </Td>
-                </Tr>
-              )}
-            </Fragment>
-          ))}
-        </Tbody>
-      </ChakraTable>
+  return (
+    <Box>
```

Comment:
Propsless `Box` should just be a `<div>`. Maybe it could just be a fragment in this case?
```suggestion
    <div>
```